components/surveyWidget/widgetBox/index.jsx

```diff
@@ -82,7 +83,8 @@ export default function WidgetBox({ id, open, anchorEl, onClose }) {
 WidgetBox.propTypes = {
   id: T.string,
   open: T.bool,
-  anchorEl: T.element,
+  // eslint-disable-next-line react/forbid-prop-types
+  anchorEl: T.any,
```

Comment:
Why wouldn't this be `T.element`? Or should it have actually be a `T.node` instead? See https://dev.to/fromaline/jsxelement-vs-reactelement-vs-reactnode-2mh2 for the difference being kind of explained

Reply:
Neither T.element or T.node work. I updated to say T.object which is more explicit but it still requires an es lint exception

Reply:
Strange! Those _should_ work if anchorEl is a jsx element.. but whatever, let's just prevent the warning & consider it debt to avoid in the future

Reply:
Strange! Those _should_ work if anchorEl is a jsx element.. but whatever, let's just prevent the warning & consider it debt to avoid in the future